src/main/java/com/stripe/param/AccountUpdateParams.java

```diff
@@ -215,14 +215,14 @@ public Builder setBusinessProfile(BusinessProfile businessProfile) {
       return this;
     }
 
-    /** The business type. Can be `individual` or `company`. */
-    public Builder setBusinessType(String businessType) {
+    /** The business type. */
+    public Builder setBusinessType(BusinessType businessType) {
```

Comment:
this might be wrong -- do we need all 3, BusinessType, String, and EmptyParam?

Reply:
(If so, https://git.corp.stripe.com/stripe-internal/pay-server/pull/198350)

Reply:
We only need `EmptyParam` in update requests if the value is unsettable, which I don't think is the case here.

Reply:
So the fact that we had it before was a mistake?

Reply:
(Re: our convo, in https://github.com/stripe-archive/stripe-api-codegen/pull/143 we made all string params emptyable to support the empty string. The empty string isn't a valid input here -- the API will reject it.

Reply:
Final conclusion: we'll keep both the `String` and `EmptyParam` overloads for now to avoid a breaking change, and remove those in the next major version.